lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.ts

```diff
@@ -154,4 +152,8 @@ export class CardViewDateItemComponent extends BaseCardView<CardViewDateItemMode
     update() {
         this.cardViewUpdateService.update({ ...this.property } as CardViewDateItemModel, this.property.value);
     }
+
+    getDateValue(newDateValue): Date {
```

Comment:
missing type for newDateValue

Reply:
we are calling our method from onDateChanged() & this method is triggering on the date change from UI. This method accepts the $event object & does not use type in this method too. We can use MatDatepickerInputEvent<any, any> type here. Is it fine?

Reply:
hi @AleksanderSklorz added type here as MatDatepickerInputEvent<any, any> for $event, also checked and found **any** as the best supported type here.

Reply:
@jatin2008 based on what information you think that `any` is the best supported type? That's not true, please go through the documentation https://material.angular.io/components/datepicker/overview especially the section with `change event` to check what type should be used here

Reply:
Btw `any` is against strongly typed TypeScript code, it's basically antipattern that we should avoid as much as we can, so please update your knowledge about best practices for TS